### PR TITLE
Made a first implementation of GraphQL. It is a working example of ho…

### LINKED ISSUE #35: Setup GRAPHQL api skeleton

### PR DESCRIPTION
**Pull request by lmtjanssen**
closes #35

### CODE REVIEW

**Review by lfb0801 — APPROVED** [review]
Project works, does however run into a warning, let's hope this gets through CI so we can fix later

File: src/main/java/com/fontys/api/ApiApplication.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 	public static void main(String[] args) {
 		SpringApplication.run(ApiApplication.class, args);
+		System.out.println("http://localhost:8080/graphiql");
```
Comment (MindHawk) [inline src/main/java/com/fontys/api/ApiApplication.java:11]: What's this for?

File: src/main/java/com/fontys/api/mutations/TeamMutation.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@Component
+public class TeamMutation implements GraphQLMutationResolver {
+
+    @Autowired
+    private TeamService teamService;
+
+    public Team createTeam(Long id, String name) {
+        return teamService.createTeam(id, name);
+    }
+}
```
Comment (MindHawk) [inline src/main/java/com/fontys/api/mutations/TeamMutation.java:18]: Don't know GraphQL, would like an explanation on what mutations are and why they're needed.

File: src/main/java/com/fontys/api/queries/TeamQuery.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    TeamService teamService;
+
+    public List<Team> getTeams(final int count) {
+        return this.teamService.getAllTeams(count);
+    }
+
+    public Optional<Team> getTeam(final Long id) {
+        return this.teamService.getTeam(id);
+    }
+}
```
Comment (MindHawk) [inline src/main/java/com/fontys/api/queries/TeamQuery.java:25]: Same as for mutation. Explain this for future reference?

**Review by MindHawk — COMMENTED** [review]
Shouldn't target/... be in git ignore?

**Review by MindHawk — APPROVED** [review]

**Pull request merged**